images/salt-master/Dockerfile

```
@@ -3,7 +3,7 @@ FROM centos:7.6.1810
 MAINTAINER moonshot-platform <moonshot-platform@scality.com>
 
 # Versions to use
-ARG SALT_VERSION=3002.2
+ARG SALT_VERSION=3002.5
```

The Salt version is already passed to the container build context from the buildchain, it should be:
```suggestion
ARG SALT_VERSION
```
This way we don't have to maintain the version in 2 places.

---

Right, it's not linked to this PR, but I agree

---

Done